### ISSUE TITLE
Motion

### ISSUE THREAD

**Issue by smygarn** [TomorrowTodayLabs/NewtonVR#28]
Great updates lately!

However, try to put the interactable objects (switch, button etc) on a moving gameobject. There may be some undesirable results to all rigidbodies. Some works better if you create fixed joints, some not.

**Comment by keithbradner** [comment]
Yeah. We're trying to figure out a solution to this. We've got an example movable button that gets triggered whenever you move it ;) But at least the constraints are local instead of global now (sideways buttons etc)

**Comment by smygarn** [comment]
A quick workaround is to constantly add the moving velocity to the objects. Which in my case, is taken from the parent rigidbody.

**Comment by keithbradner** [comment]
Update: Yeah, not sure the best solution to this still.
